src/appState.ts

```diff
@@ -237,6 +237,11 @@ export class AppState implements MutationManager {
                         const match = /^[-+]?\d+/.exec(givenInput);
```

Comment:
```suggestion
                        const match = /^[-+]?\d+\s*/.exec(givenInput);
```

확실히 `1 2 3` 같은 코드를 `방방방`으로 읽고싶을 수는 있을 것 같으니, 공백을 구분자로 사용하도록 하는 것은 어떤가 싶습니다.